app/templates/gentelella/users/events/_table.html

```diff
@@ -39,7 +39,7 @@
                 {% endfor %}
 
                 <tr>
-                    <td>  <img src="{{ event.logo }}"  onerror="imgError(this); " style="height:60px ; width:60px;"><a href="{{ url_for('events.details_view', event_id=event.id) }}"><br>&ensp;{{ event.name }}</a></td>
+                    <td><a href="{{ url_for('events.details_view', event_id=event.id) }}"><img src="{{ event.logo }}" style="width: 7rem; height: 5rem;" onerror="imgError(this);"><p style ="padding:8px;">{{ event.name }}</p></a></td>
```

Comment:
All that is supposed to be in this PR is this line change. Nothing else. Could you please do that ?